src/ui/buttons/buttonDesigns/index.js

```diff
@@ -31,6 +43,30 @@ const DESIGN_MAP : Object = {
         getValidDesignProps:   getValidInlineLogoTextProps,
         designConfig:          INLINE_LOGO_TEXT_CONFIG,
         ButtonDesignComponent: InlineLogoTextComponent
+    },
+    'run-switch-logo-show-label-text-design': {
+        designFn:              getSwitchLogoAndShowLabelAnimation,
+        getValidDesignProps:   getSwitchLogoAndShowLabelProps,
+        designConfig:          SWITCH_LOGO_AND_SHOW_LABEL_CONFIG,
+        ButtonDesignComponent: SwitchLogoAndShowLabelTextComponent
+    },
+    'run-switch-logo-show-label-text-once-design': {
+        designFn:              getSwitchLogoAndShowLabelAnimation,
+        getValidDesignProps:   getSwitchLogoAndShowLabelProps,
+        designConfig:          { ...SWITCH_LOGO_AND_SHOW_LABEL_CONFIG, runOnce: true },
+        ButtonDesignComponent: SwitchLogoAndShowLabelTextComponent
+    },
+    'run-resize-button-and-show-label-design': {
+        designFn:              resizeButtonShowLabelTextAnimation,
+        getValidDesignProps:   resizeButtonShowLabelTextProps,
+        designConfig:          DESIGN_SMALL_BUTTON_CONFIG,
+        ButtonDesignComponent: resizeButtonShowLabelTextComponent
+    },
+    'run-resize-button-and-show-label-once-design': {
+        designFn:              resizeButtonShowLabelTextAnimation,
+        getValidDesignProps:   resizeButtonShowLabelTextProps,
+        designConfig:          { ...DESIGN_SMALL_BUTTON_CONFIG, runOnce: true },
+        ButtonDesignComponent: resizeButtonShowLabelTextComponent
     }
```

Comment:
Can we switch this to `reveal-tagline-layer-single`

Reply:
done